src/output_lib/components/RespondToForm/Validators/EmailValidator.js

```diff
@@ -0,0 +1,8 @@
+function valid(email) {
+  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
move this const outside of the function, so it's only computed once. Otherwise we're building a static regex on the fly each time the function is invoked.